index.html

```diff
@@ -1,29 +1,29 @@
 <!DOCTYPE html>
-<html lang=“ja”>
+<html lang="en">
   <head>
-    <meta charset=“UTF-8" />
-    <meta name=“viewport” content=“width=device-width, initial-scale=1.0" />
-    <meta http-equiv=“X-UA-Compatible” content=“ie=edge” />
-    <link rel=“stylesheet” href=“style.css” type=“text/css” />
-    <title>TerraceTechフロントエンドエンジニア養成所(I used $i）</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
-      <div>
-        <ul id="js-list"></ul>
-      </div>
+    <div>
+      <ul id="js-list"></ul>
+    </div>
   <script>   
     const ul = document.getElementById('js-list');
-      for(let i=1; i<=2; i++) {
-      const li = document.createElement('li');
-      ul.appendChild(li);
-      const a = document.createElement('a');
-      a.textContent = `a${i}`;
-      a.href= `a${i}.html`;
-      li.appendChild(a);
-      const img = document.createElement('img');
-      img.src = '/img/bookmark.png';
-      a.insertBefore(img, a.firstChild);
+    for(let i=1; i<=2; i++) {
+    const li = document.createElement('li');
+    ul.appendChild(li);
+    const a = document.createElement('a');
+    a.textContent = `a${i}`;
+    a.href= `a${i}.html`;
+    li.appendChild(a);
+    const img = document.createElement('img');
+    img.src = '/img/bookmark.png';
+    a.insertBefore(img, a.firstChild);
```

Comment:
```
ul.appendChild(li);
li.appendChild(a);
a.insertBefore(img, a.firstChild);
```

Furthermore, in this source, the DOM is accessed every time multiple elements are inserted.
This is problematic from a performance standpoint, and I was also pointed out that using DocumentFragment is a good idea, so please refer to it.

> const fragment = document.createDocumentFragment();
> { // inside the loop
> fragment.appendChild(li).appendChild(a).insertBefore(img, a.firstChild);
> }
> //add at the end
> ul.appendChild(fragment)

Reply:
I didn't know that adding it every time would cause performance issues.
Thanks for letting me know about DocumentFragmen.
I will make use of it in the future.